web/submissions_test.go

```diff
@@ -193,13 +193,13 @@ func TestSubmissionsAccess(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	haveSubmissions, err = ags.GetSubmissions(ctx, &pb.SubmissionRequest{CourseID: course.ID})
+	gotSubmissions, err = ags.GetSubmissions(ctx, &pb.SubmissionRequest{CourseID: course.ID})
 	if err != nil {
 		t.Fatal(err)
 	}
 	// enrolled as student, admin must be able to access all course submissions
-	if !reflect.DeepEqual(latestSubmissions, haveSubmissions.GetSubmissions()) {
-		t.Errorf("Admin got submissions %+v, expected submissions: %+v", haveSubmissions.GetSubmissions(), latestSubmissions)
+	if diff := cmp.Diff(gotSubmissions.GetSubmissions(), latestSubmissions, protocmp.Transform()); diff != "" {
+		t.Errorf("TestSubmissionsAccess() mismatch (-gotSubmission.GetSubmissions(), +latestSubmissions):\n%s", diff)
```

Comment:
I would prefer to not have `.GetSubmissions` in the string like this; what about something like this:
```go
submissions, err = ags.GetSubmissions(ctx, &pb.SubmissionRequest{CourseID: course.ID})
if err != nil {
	t.Fatal(err)
}
// enrolled as student, admin must be able to access all course submissions
gotSubmissions := submissions.GetSubmissions()
if diff := cmp.Diff(gotSubmissions, latestSubmissions, protocmp.Transform()); diff != "" {
	t.Errorf("ags.GetSubmissions() mismatch (-gotSubmissions, +latestSubmissions):\n%s", diff)
}
```